onnxruntime/core/session/ort_env.cc

```diff
@@ -37,6 +37,11 @@ OrtEnv* OrtEnv::GetInstance(const OrtEnv::LoggingManagerConstructionInfo& lm_inf
                             onnxruntime::common::Status& status,
                             const OrtThreadingOptions* tp_options) {
   std::lock_guard<onnxruntime::OrtMutex> lock(m_);
+  if (p_instance_) {
+    ++ref_count_;
+    return p_instance_;
+  }
+
```

Comment:
Could you instead move the code to create the logging manager inside the 'if (!p_instance_)' section? Slightly less code that way.

Reply:
Done